.github/pull_request_template.md

```diff
@@ -0,0 +1,51 @@
+---
+name: "New Pull Request (PR)"
+about: For including new features into the ontology
+title: Your title should make sense if said after "This PR is solving <your issue>"
+labels: "Pull Request Template"
+assignees: ''
+
+---
+
+
+## Summary of the discussion
+
+Here describe the findings of the discussion in the issue or meeting.
+
+## Type of change
+
+### Added
+- Added a new class [#](https://github.com/OpenEnergyPlatform/ontology/issues/)
+
+### Updated
+- Updated a definition [#](https://github.com/OpenEnergyPlatform/ontology/issues/)
+
+### Removed
+- Removed a broken link [#](https://github.com/OpenEnergyPlatform/ontology/issues/)
+
+
+## Workflow checklist
+
+### Assignee
+- [ ] An agreement has been reached
+- [ ] 🐙 Add yourself as Assignee
+- [ ] 🐙 Add someone or a group (oeo-dev) as Reviewer
```

Comment:
"reviewer assignment" should be done after switching from Draft-PR to PR. 
Checking yesterdays protocol, I noticed we wrote it there, too.

Had to use the template in order to notice it :D